<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VibeTube - Home Page</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget {
    background-color: #FFE6F0;
    color: black; 
}
QPushButton {
    background-color: #FF6F91;
    color: white;
    font-weight: bold;
    border-radius: 8px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #FF9AA2;
}
QLineEdit, QComboBox {
    background-color: #FFFFFF;
    border: 2px solid #FFC1CC;
    border-radius: 6px;
    padding: 4px;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold; color: black;</string>
     </property>
     <property name="text">
      <string>Viewer Home</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchInput">
     <property name="placeholderText">
      <string>Search videos...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTrending">
     <property name="title">
      <string>Trending Videos</string>
     </property>
     <layout class="QVBoxLayout" name="layoutTrending">
      <item>
       <widget class="QListWidget" name="listTrending"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnLogout">
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
